src/Tizen.NUI/src/internal/Common/WeakEvent.cs

```diff
@@ -106,13 +124,46 @@ public WeakHandler(U handler)
                 methodInfo = d.Method;
             }
 
+            public bool Equals(object? otherTarget, MethodInfo otherMethodInfo)
```

Comment:
```suggestion
            private bool EqualsByTargetObject(object? otherTarget, MethodInfo otherMethodInfo)
```

Maybe we need to make it as private + change the name?

Reply:
I removed IEquatable related implementation because the code do not use HashSet anymore.